docs/source/config_file.rst

```diff
@@ -333,6 +333,11 @@ a list of import discovery options that may be used
     Specifies the paths to use, after trying the paths from ``MYPYPATH`` environment
     variable.  Useful if you'd like to keep stubs in your repo, along with the config file.
 
+``files`` (string)
+    Specifies the paths which should be checked by mypy if none are given on the command
+    line. Supports recursive file globbing with `*` (eg `*.py` for files in the same directory)
```

Comment:
I think it might be good to add we pass globbing onto `glob`  (with `recursive=True`) and link https://docs.python.org/3/library/glob.html as glob supports more than `*` and `**`, though those are likely the most used constructs.